security/fma-incident-response-improvements.md

```diff
@@ -0,0 +1,246 @@
+# Incident Response Improvements: Failure Modes and Recovery Path Analysis
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Incident Response Improvements: Failure Modes and Recovery Path Analysis](#incident-response-improvements-failure-modes-and-recovery-path-analysis)
+  - [Introduction](#introduction)
+  - [FM1: Contract deployment and/or initialization failures](#fm1-contract-deployment-andor-initialization-failures)
+    - [Description](#description)
+    - [Risk Assessment](#risk-assessment)
+    - [Mitigations](#mitigations)
+    - [Detection](#detection)
+    - [Recovery Path(s)](#recovery-paths)
+  - [FM2: Legacy runbook/response confusion](#fm2-legacy-runbookresponse-confusion)
+    - [Description](#description-1)
+    - [Risk Assessment](#risk-assessment-1)
+    - [Mitigations](#mitigations-1)
+    - [Detection](#detection-1)
+    - [Recovery Path(s)](#recovery-paths-1)
+    - [Action items](#action-items)
+  - [FM3: Anchor state fails to progress within a time bound](#fm3-anchor-state-fails-to-progress-within-a-time-bound)
+    - [Description](#description-2)
+    - [Risk Assessment](#risk-assessment-2)
+      - [Mitigations](#mitigations-2)
+      - [Detection](#detection-2)
+      - [Recovery Path(s)](#recovery-paths-2)
+      - [Action items:](#action-items-1)
+  - [FM4: Errors in bond distribution](#fm4-errors-in-bond-distribution)
+    - [Description](#description-3)
+    - [Risk Assessment](#risk-assessment-3)
+    - [Mitigations](#mitigations-3)
+    - [Detection](#detection-3)
+    - [Recovery Path(s)](#recovery-paths-3)
+    - [Action items](#action-items-2)
+    - [Generic items we need to take into account:](#generic-items-we-need-to-take-into-account)
+  - [Action Items](#action-items-3)
+  - [Audit Requirements](#audit-requirements)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+_Italics are used to indicate things that need to be replaced._
+
+|                    |                |
+| ------------------ | -------------- |
+| Author             | wildmolasses   |
+| Created at         | 2025-01-22     |
+| Initial Reviewers  | smartcontracts |
+| Need Approval From |                |
+| Status             | Draft          |
```

Comment:
I'm assuming this is actually ready for review? 
```suggestion
| Status             | In Review          |
```

Reply:
Resolved in 855cf68